thread/stack-allocator.cpp

```diff
@@ -96,9 +96,10 @@ class PooledStackAllocator {
             return __alloc(size);
         }
         auto ptr = slots[idx].get();
+        auto aligned_size = MIN_ALLOCATION_SIZE << idx;
```

Comment:
why not put it in the scope of the following if

Reply:
seems good